L1Trigger/L1TTrackMatch/BuildFile.xml

```diff
@@ -10,6 +10,8 @@
 <use   name="root"/>
 <use   name="rootrflx"/>
 <use   name="hls"/>
+<use   name="conifer"/>
+<use   name="hepmc"/>
```

Comment:
Is this used?

Reply:
The CI was failing without these additions so they're necessary

Reply:
I'd say the reason for why `hepmc` seems to be needed should be understood first. L1Trigger code depending on HepMC sounds strange.

Ok, I see
https://github.com/cms-sw/cmssw/blob/96923256cad2e4d3494057a8b8d2c2657f57b99b/L1Trigger/L1TTrackMatch/test/L1TkMETAnalyser.cc#L9
but it doesn't seem to be used. If it would be, the proper place to add the dependence would be in
https://github.com/cms-sw/cmssw/blob/master/L1Trigger/L1TTrackMatch/test/BuildFile.xml

Reply:
The plugins already depend on HepMC from the master branch build file: https://github.com/cms-sw/cmssw/blob/master/L1Trigger/L1TTrackMatch/plugins/BuildFile.xml. The only difference is my includes, which need HepMC, are in a header file which uses the build file here https://github.com/cms-sw/cmssw/blob/master/L1Trigger/L1TTrackMatch/BuildFile.xml

Reply:
I believe the include that needs HepMC is the TTtrack association here: https://github.com/ryanm124/cmssw/blob/f36f6162982a95dcd5b7acfa6609220e4855ac9b/L1Trigger/L1TTrackMatch/interface/DisplacedVertexProducer.h#L14